### PR TITLE
Remove upload fallback cx 2297

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Live Capture should be enforced whenever possible.

# Solution
Remove upload fallback link from camera screen.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [x] Have new manual tests been written down?
- [x] Have tests passed locally?
- [x] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-369-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://369-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/index.js [hunk 1]
```diff
@@ -65,8 +52,8 @@ export class CameraPure extends React.Component<CameraPureType> {
   }
 
   render() {
-    const {method, title, subTitle, onUploadFallback, onFallbackClick, hasError,
-      onUserMedia, onFailure, webcamRef, isFullScreen, i18n, video} = this.props;
+    const {method, title, subTitle, hasError, onUserMedia,
```
Comment (rfreitas) [inline src/components/Camera/index.js]: the `onUploadFallback` is coming from the `capture` component. Logic there should also be removed.

Comment (stefaniacardenas) [reply src/components/Camera/index.js]: I'm going to need that logic in this PR #372 which is pointing at this branch

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-369-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-369-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://369-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**